Upstream-Status: Inappropriate [oe specific]

From ecdf13e6158002f3beb66aa4bdfee85e683cd59d Mon Sep 17 00:00:00 2001
From: customer <customer@company.com>
Date: Fri, 18 Apr 2025 00:09:02 +0100
Subject: [PATCH] auto-program-eeprom-on-first-boot

---
 board/ti/common/board_detect.c | 3 ++-
 1 file changed, 2 insertions(+), 1 deletion(-)

diff --git a/board/ti/common/board_detect.c b/board/ti/common/board_detect.c
index 38e23ccbb67..a49c0abbc87 100644
--- a/board/ti/common/board_detect.c
+++ b/board/ti/common/board_detect.c
@@ -320,7 +320,8 @@ int __maybe_unused ti_i2c_eeprom_am_get(int bus_addr, int dev_addr)
 	rc = ti_i2c_eeprom_get(bus_addr, dev_addr, TI_EEPROM_HEADER_MAGIC,
 			       sizeof(am_ep), (uint8_t *)&am_ep);
 	if (rc)
-		return rc;
+        	puts ("\nUse hard coded Board Name and Revision in case the EEPROM is not programmed.\n");
+	return ti_i2c_eeprom_am_set("A335BNLT", "00C0");
 
 	ep->header = am_ep.header;
 	strlcpy(ep->name, am_ep.name, TI_EEPROM_HDR_NAME_LEN + 1);
-- 
2.43.0

