Upstream-Status: Inappropriate [oe specific] 

From 3dc383d9f90ab03c79130100c7cb3f654f334027 Mon Sep 17 00:00:00 2001
From: customer <customer@company.com>
Date: Fri, 18 Apr 2025 19:39:05 +0100
Subject: [PATCH] trap-blank-eeprom-uboot-board-detect

---
 board/ti/common/board_detect.c | 23 +++++++++++++++++++++++
 1 file changed, 23 insertions(+)

diff --git a/board/ti/common/board_detect.c b/board/ti/common/board_detect.c
index 38e23ccbb67..c1cb6a73273 100644
--- a/board/ti/common/board_detect.c
+++ b/board/ti/common/board_detect.c
@@ -319,6 +319,29 @@ int __maybe_unused ti_i2c_eeprom_am_get(int bus_addr, int dev_addr)
 
 	rc = ti_i2c_eeprom_get(bus_addr, dev_addr, TI_EEPROM_HEADER_MAGIC,
 			       sizeof(am_ep), (uint8_t *)&am_ep);
+
+	/* 
+	 *     Derived from (c) 2025 Robert Nelson
+	 *     This is taken from the beaglebone.org repository to keep things
+	 *     consistent.  Detects a blank eeprom or malformed code
+	 *     This is for patching POKY not Beagleboard version
+	 */
+
+	if (rc < 0) {
+
+		puts ("\Blank or malformed eeprom code\n");
+
+		ep->header = TI_EEPROM_HEADER_MAGIC;
+
+		strlcpy(ep->name, "A335BLNK", TI_EEPROM_HDR_NAME_LEN + 1);
+		ti_eeprom_string_cleanup(ep->name);
+
+		strlcpy(ep->version, "EMMC", TI_EEPROM_HDR_REV_LEN + 1);
+		ti_eeprom_string_cleanup(ep->version);
+
+		return 0;
+	}
+
 	if (rc)
 		return rc;
 
-- 
2.43.0

