Upstream-Status: Inappropriate [oe specific]

From 13c32f0cb5a8c82ff0e46f43250c20907e7c0faf Mon Sep 17 00:00:00 2001
From: customer <customer@company.com>
Date: Fri, 11 Apr 2025 16:53:39 +0100
Subject: [PATCH] Identify new boards

---
 board/ti/common/board_detect.c | 7 +++++--
 1 file changed, 5 insertions(+), 2 deletions(-)

diff --git a/board/ti/common/board_detect.c b/board/ti/common/board_detect.c
index 38e23ccbb67..cec51261d88 100644
--- a/board/ti/common/board_detect.c
+++ b/board/ti/common/board_detect.c
@@ -319,8 +319,11 @@ int __maybe_unused ti_i2c_eeprom_am_get(int bus_addr, int dev_addr)
 
 	rc = ti_i2c_eeprom_get(bus_addr, dev_addr, TI_EEPROM_HEADER_MAGIC,
 			       sizeof(am_ep), (uint8_t *)&am_ep);
-	if (rc)
-		return rc;
+	
+	if (rc || am_ep.header != TI_EEPROM_HEADER_MAGIC) {
+               printf("New Instrument, storing default BBB Rev C\n");
+               return ti_i2c_eeprom_am_set("A335BNLT","C");
+        }
 
 	ep->header = am_ep.header;
 	strlcpy(ep->name, am_ep.name, TI_EEPROM_HDR_NAME_LEN + 1);
-- 
2.43.0

