Upstream-Status: Inappropriate [oe specific] 

From 7917e376ef0a4b737a2df1ae27699f3fa585c033 Mon Sep 17 00:00:00 2001
From: customer <customer@company.com>
Date: Fri, 18 Apr 2025 21:20:34 +0100
Subject: [PATCH] trap-blank-eeprom-uboot-am335x_evm

---
 include/configs/am335x_evm.h | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/include/configs/am335x_evm.h b/include/configs/am335x_evm.h
index 67f46562539..19a1d1bde08 100644
--- a/include/configs/am335x_evm.h
+++ b/include/configs/am335x_evm.h
@@ -118,6 +118,8 @@
 			"setenv fdtfile am335x-bone.dtb; fi; " \
 		"if test $board_name = A335BNLT; then " \
 			"setenv fdtfile am335x-boneblack.dtb; fi; " \
+		"if test $board_name = A335BLNK; then " \
+			"setenv fdtfile am335x-boneblack.dtb; fi; " \
 		"if test $board_name = A335PBGL; then " \
 			"setenv fdtfile am335x-pocketbeagle.dtb; fi; " \
 		"if test $board_name = BBBW; then " \
-- 
2.43.0

